src/storage-client/src/controller.rs

```diff
@@ -1435,7 +1449,22 @@ where
                     metadata.clone(),
                 );
 
-                self.state.persist_write_handles.register(id, write);
+                match description.data_source {
+                    DataSource::Introspection(_) | DataSource::Webhook => {
+                        debug!(desc = ?description, meta = ?metadata, "registering {} as monotonic", id);
```

Comment:
nit: these logs out of context are a little unclear to me. How would you feel about something like...

- `registering {} with persist monotonic worker`
- `registering {} with persist table worker`
- `not registering {} with a controller persist worker`

Reply:
done!